builds.yml

```diff
@@ -278,6 +278,8 @@ env:
     # Enables the notifications feature within the build:
   - NOTIFICATIONS: ''
 
+  - METAMASK_RAMP_API_BASE_URL: https://on-ramp-content.metaswap.codefi.network
```

Comment:
Let's rename this to `METAMASK_RAMP_API_CONTENT_BASE_URL` since this is the content endpoint and not the  regular one we use for order tracking for example (`https://on-ramp.api.cx.metamask.io/...`), the value must be updated to `https://on-ramp-content.api.cx.metamask.io`

Reply:
988e98509c129d77fb4127cbed2f94b974ee4718